docs/Behaviour - Client Side.md

```diff
@@ -2,7 +2,17 @@
+Before making an [IS-05][IS-05] connection, an NMOS Controller chooses a Sender and a number of Receivers. The NMOS Controller MAY keep the current configuration of the Sender or change its configuration. In case of changing configuration of an active Sender, deactivating such Sender MAY be required and depends on the Sender itself.
+
+If changing the configuration of the Sender, the NMOS Controller MUST `GET /constraints/supported` from the Sender, MUST collect Receiver Capabilities from the Receivers and make such processing of them that they can be utilized for building Active Constraints satisfying all (by default) or some (depending on user preferences) of the Receivers and using Parameter Constraint URNs supported by the Sender. If the Sender supports fewer Parameter Constraint URNs than used in the Receiver Capabilities, the NMOS Controller SHOULD inform the user about it. If the processing of the Receiver Capabilities results in an empty `constraint_sets` of Active Constraints, the NMOS Controller SHOULD inform the user that all of the specified Receivers can't be satisfied at the same time. After that the NMOS Controller MUST `PUT /constraints/active` to the Sender and SHOULD make the IS-05 connections of the Receivers if the Constraints have been applied succesfully to the Sender.
+
+If keeping the configuration of the Sender, the NMOS Controller MUST `GET /constraints/supported` and `GET /constraints/active` from the Sender, MUST collect Receiver Capabilities from the Receivers and make such processing of them that they can be utilized for building Active Constraints satisfying all (by default) or some (depending on user preferences) of the Receivers and using Parameter Constraint URNs supported by the Sender. If the Sender supports fewer Parameter Constraint URNs than used in the Receiver Capabilities, the NMOS Controller SHOULD inform the user about it. If the processing of the Receiver Capabilities results in an empty `constraint_sets` of Active Constraints, the NMOS Controller SHOULD inform the user about it.
+Sometimes there can be streams compliant with the current active constraints of the Sender, but not compliant with the resulting `constraint_sets`. In these scenarios the NMOS Controller SHOULD inform the user that the Receivers cannot be accommodated without replacing the Active Constraints.
+After that the NMOS Controller SHOULD make the IS-05 connection of the Receivers.
```

Comment:
Do we really want a Controller to go ahead and make the connections if it has identified that some of the Receivers cannot be accommodated? Do we want to recommend that the User is involved at this point? (Note that in the "changing the configuration" scenario, the Controller SHOULD make the connection if the constraints have been successfully applied. Why should it be different in the "keeping the configuration" case?)

Reply:
```suggestion

After this the NMOS Controller can make the chosen connections via IS-05.
```